test/FunctionalTests/Microsoft.OData.Edm.Tests/Validation/ValidationRulesTests.cs

```diff
@@ -1491,7 +1510,7 @@ internal TestFunction(string namespaceName, string name)
 
             public IEdmOperationParameter FindParameter(string name)
             {
-               return this.Parameters.Single(p=>p.Name == name);
+                return this.Parameters.Single(p => p.Name == name);

```

Comment:
nit: remove whitespace

Reply:
this formats the line to be consitent in the file.

Reply:
this reformats this line to be more consistent in the file.